wiki/docs/data-requirements.md

```diff
@@ -15,9 +15,9 @@ Users who would like to model a local energy market (an energy community partici
     * **Storage (Battery)**
         * Name (can be anonymous, e.g. Battery1) and location
         * Capacity in kWh
-        * Power delivery in kW
-        * Maximum charge/discharge rate per time interval in kW
-        * Minimum SOC in % and/or actual SOC per time interval (optional)
+        * Charge / discharge power in kW
```

Comment:
This is not an input parameter of the strategy, but rather the historical energy production / consumption profile of the battery. We have never had it from any pilot site till now, but we can potentially use this for our base case scenaria. 
If the base case includes batteries, we use the normal StorageStrategy, which is kind of unrealistic because we do not replicate the actual behavior of the battery at the time of the simulation. Contrary to that, it is way more realistic to model the battery as a load for the base case scenario, and to use the StorageStrategy for the simulated scenaria where we use the extra flexibility provided by it. That was the rationale behind including it some years ago, and I think it still makes sense to keep, even if we very rarely have it available.

Reply:
Ok, thank you for the clarification!